examples/visual-art-making-tools/basic/example.js

```diff
@@ -0,0 +1,158 @@
+//BASIC EXAMPLE 
+// three colors red, yellow, blue
+//floor image of wood
+//access colors using r g b variables
+
+const brushStrokes = [];
+const maxOrbs = 500;
+let img;
+let ray;
+
+let r = 0;
+let g = 0;
+let b = 0;
+let valSize = 0.05;
+
+let uiForward = 2;
+let uiLeft = -0.75;
+let uiRight = 0.75;
+let uiTop = 0.95;
+let uiMiddle = 0.5;
+let uiBottom = 0.15;
+let uiSize = 0.15;
+
+function preload() {
+  createVRCanvas();
+img = loadImage('woodtex.jpg')
+}
+
+function setup() {
+  setVRBackgroundColor(235, 230, 235);
+  angleMode(DEGREES);
+}
+
+function draw() {
+
+  drawGround();
+
+  const left = getXRInput(LEFT);
+  const right = getXRInput(RIGHT);
+  let hand;
+
+  if (right) {
+    hand = right;
+  } else if (left) {
+    hand = left;
+  } else {
+    // no controller
+  }
+
+  if (hand) {
+    if (hand.trigger && hand.trigger.pressed) {
+      attemptNewBrushStroke(hand);
+    }
+
+    // Apply the pose matrix for the hand
+    // Create a ray, note that rays are made with the current
+    // model view matrix applied so it is easier to assign
+    // position and directionality because it is relative to the
+    // current transform, just like rendering
```

Comment:
Maybe break this out into its own function so that it is easier to see what this code is doing.